### DISCUSSION TITLE
400 Bad Request when setting up nextcloud aio

### DISCUSSION THREAD

**Discussion by Hoshikawakaguya** [nextcloud/all-in-one#3275] [Questions] [answered]
I can use the domain:8080 to access the setup page, but when I submit the domain, it shows
Domain does not point to this server or the reverse proxy is not configured correctly. See the mastercontainer logs for more details. ('sudo docker logs -f nextcloud-aio-mastercontainer')
The log is 
```
NOTICE: PHP message: Expected was: 991511a14fd94f2e2d604a4e1d7633ef4c5a750d0a0afb4a
NOTICE: PHP message: The error message was:
NOTICE: PHP message: The response of the connection attempt to "http://mydomain.domain:443" was:
                               <HTML>                                              <HEAD><TITLE>400 Bad Request</TITLE></HEAD>                                              <BODY BGCOLOR="#FFFFFF" TEXT="#000000" LINK="#2020ff" VLINK="#4040cc">                                              <H2>400 Bad Request</H2>Your request has bad syntax or is inherently impossible to satisfy.</body></html>
```
If I just skip the domain check, I will finally get ERR_SSL_PROTOCOL_ERROR

However, I can access http://mydomain.domain:443 myself, I will get a random string.

**Comment by szaimen** [comment, marked as answer]
> If I just skip the domain check, I will finally get ERR_SSL_PROTOCOL_ERROR

Can you follow https://github.com/nextcloud/all-in-one/discussions/2105?
